res/css/views/emojipicker/_EmojiPicker.scss

```diff
@@ -32,6 +32,7 @@ limitations under the License.
 }
 
 .mx_EmojiPicker_header {
+    display: inherit;
```

Comment:
usually this means something went disastrously wrong with the layout - what is this counteracting?